ansible_ai_connect/users/reports/tests/test_generators.py

```diff
@@ -209,10 +209,7 @@ def tearDown(self):
         super().cleanup()
 
     def get_report_header(self) -> str:
-        return (
-            "OrgId,UUID,First name,Last name,Email,Organization name,"
-            "Plan name,Trial started,Trial expired_at"
-        )
+        return "OrgId,UUID,First name,Last name,Organization name,Email," "Plan name,Trial started"
```

Comment:
You can concatenate this into a single string.

I suspect re-formatting removed the tuple but didn't join the strings.

Reply:
Unfortunately the line becomes too long and Flake8 will insult me.